src/playback/iplaybackcontroller.h

```diff
@@ -101,6 +101,8 @@ class IPlaybackController : MODULE_EXPORT_INTERFACE
 
     virtual void setNotation(notation::INotationPtr notation) = 0;
     virtual void setIsExportingAudio(bool exporting) = 0;
+
+    virtual void toggleHearPlaybackWhenEditing() = 0;
```

Comment:
As long as this is used only via the action code, it doesn't need to be part of the interface